python/ray/llm/_internal/batch/stages/vllm_engine_stage.py

```diff
@@ -678,26 +684,42 @@ def post_init(cls, values):
         # Ray Data won't reserve GPUs in advance. Instead, we specify scheduling
         # strategy in .map_batches() arguments and let vLLM Ray executor to
         # create placement groups for each TP/PP worker.
-        resources_per_bundle = map_batches_kwargs.pop("resources", None)
+        placement_group_config = fn_constructor_kwargs.pop(
+            "placement_group_config", None
+        )
         if executor_backend == "ray" and num_bundles_per_replica > 1:
```

Comment:
I have a question? should `num_bundles_per_replica > ` be a gating condition here? What if it is 1 in case of TP=1, PP=1?

Reply:
Also I think in a follow up PR after this we should default to "ray" for backend

@nrghosh

Reply:
actually I think we should do it in this PR and add the TP=1, PP=1 to the tests.